frontend/package.json

```diff
@@ -45,6 +45,7 @@
     "clsx": "^2.0.0",
     "cmdk": "^0.2.0",
     "d3-format": "^3.1.0",
+    "d3-time-format": "^4.1.0",
```

Comment:
Fix version, please

Reply:
@andresgnlez Can you clarify?

Reply:
No caveat. Running `yarn add -E d3-time-format` will remove the caveat and fix the version